# Sample of motor heating information
time    stamp
float64 winding_temperature
float64 housing_temperature
float64 ambient_temperature
float64 heating_power